generated/ruby/grpc-google-iam-v1/grpc-google-iam-v1.gemspec

```diff
@@ -0,0 +1,26 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'grpc-google-iam-v1'
+  s.version       = '0.6.8'
```

Comment:
Okay, then I approve the patch as-is. You could submit as 0.1.0, or you could make this back to 0.6.8 or something.